src/lib/seam/components/ClimateSettingScheduleDetails/ClimateSettingScheduleDetails.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Dialog } from '@mui/material'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { ClimateSettingScheduleDetails } from 'lib/seam/components/ClimateSettingScheduleDetails/ClimateSettingScheduleDetails.js'
+import { useToggle } from 'lib/ui/use-toggle.js'
+
+/**
+ * These stories showcase access code details.
```

Comment:
Could remove this comment

Reply:
So these comments actually render in the stories, but they keep getting copy pasted and uncaught in reviews: 
<img width="277" alt="image" src="https://github.com/seamapi/react/assets/721372/ccb94cc0-863e-4609-8b01-3aabc7592c97">


(Good catch this time!)

Maybe we should just remove all of them? They don't add much to the stories unless we expand on them IMO.

Reply:
Oh that's cool—didn't know Storybook rendered those. I think consistency is better—I haven't been adding them.

Reply:
[6e7d9a3](https://github.com/seamapi/react/pull/375/commits/6e7d9a3e75e866d750fbe66b1c2e0fa3a2ac5bea)